stopcovid/status/users.py

```diff
@@ -24,7 +24,7 @@
 from sqlalchemy.dialects.postgresql import UUID, JSONB
 from sqlalchemy.exc import DatabaseError
 
-from . import db
+from stopcovid import db
```

Comment:
I'm cool with this, though we may end up splitting the databases out for each context. The message log and each repository could be in separate databases and they may be during the migration away from serverless aurora.